build.gradle

```diff
@@ -5,8 +5,8 @@ buildscript {
         sample_app_version = '1.0'
         sample_app_code_version = 1
 
-        omise_sdk_version = '4.11.0'
-        omise_sdk_code_version = 43
+        omise_sdk_version = '4.12.0'
+        omise_sdk_code_version = 44
```

Comment:
@AnasNaouchi what's the purpose of `omise_sdk_code_version`?

Reply:
o i c https://developer.android.com/studio/publish/versioning

Reply:
It is a general usage in the java space where `omise_sdk_code_version ` is strictly an int that is incremented and can be modified without a necessity to update `omise_sdk_version ` . Basically it can be used to do silent updates like you release 4.12.0 again . `omise_sdk_code_version ` must always be higher in number then the previous version. It also can be used to force users to update if they have a version older that what is currently distributed. 
`omise_sdk_version ` => public usage
`omise_sdk_code_version ` => Internal usage & Dev centric